### ISSUE TITLE
`regolith-session-sway` should not depend on `ilia`

### ISSUE THREAD

**Issue by gogsbread** [regolith-linux/regolith-desktop#952]
**Describe the bug**
`regolith-session-sway` transitively depends on `ilia`, through `regolith-sway-root-config` which is not correct. Some of us don't want to use ilia and this dependency makes it impossible to uninstall ilia without uninstalling entire sway

**To Reproduce**
```
> apt-rdepends regolith-session-sway
regolith-session-sway
  ...
  Depends: regolith-sway-root-config
regolith-sway-root-config 
  ...
  Depends: regolith-wm-rofication-ilia
regolith-wm-rofication-ilia
  ...
  Depends: ilia
```

Contrast this with `regolith-session-flashback` which doesn't depend on any notification manager
```
> apt-rdepends regolith-session-flashback
...
regolith-i3-root-config
  Depends: regolith-i3-default-style
  Depends: regolith-i3-session
  Depends: regolith-wm-config
```
**Expected behavior**
`regolith-session*` should not depend on `ilia`. We can have this a default behavior but please provide flexibility to install alternatives. This is similar to https://github.com/regolith-linux/regolith-desktop/issues/911, that I raised earlier.

**Comment by kgilmer** [comment]
hi @gogsbread , yes this has been resolved and will ship in the next release.  If you'd like it sooner you can install updates from the `testing` stage.

```
$ apt-rdepends regolith-session-sway | grep ilia
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
```

**Comment by austinweisgrau** [comment]
I'm not sure if this is the exact same issue, but uninstalling `regolith-sway-session` (in order to customize keybindings) also uninstalls `regolith-session-sway`. Should I make a separate issue for that?

**Comment by kgilmer** [comment]
Hi @austinweisgrau , no worries on a new issue.  Regarding your customization needs, just checking, can it not be done simply by specifying an override?

**Comment by austinweisgrau** [comment]
After your comment and a little digging, I realized I could specify a new keybinding by throwing a new file in `~/.config/regolith3/sway/config.d/`. Based on a misreading of the documentation, I thought I needed to specifically copy the config partial for session keybindings from `/usr/share/regolith3/sway/config.d/55_session_keybindings` into the config dir and then uninstall the associated subpackage. But that's not actually necessary for specifying new keybindings (vs. messing with the existing session keybindings)
